django_tasks/backends/database/models.py

```diff
@@ -167,6 +167,7 @@ def task_result(self) -> "TaskResult[T]":
 
         object.__setattr__(task_result, "_exception_class", exception_class)
         object.__setattr__(task_result, "_traceback", self.traceback or None)
+        object.__setattr__(task_result, "_return_value", self.return_value)
```

Comment:
When I rebased my changes, I noticed that this assertion was failing: https://github.com/RealOrangeOne/django-tasks/blob/dfb443efa8a5d125171783355ba2f5bd6a291db9/tests/tests/test_database_backend.py#L979

I'm pretty sure this used to work, and it looks like the database backend is not fetching return values. Did I find a bug? I'm happy to write a specific test for checking that `return_value` is set on